### PR TITLE
0.2.2 changelog

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,81 @@
...
+  + updated to ember 1.11.0
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + `ember init` once again works inside an addon.
+  + error live-reloading now actually works!
+  + npm WARN for `makeError` and `tmpl` have been fixed
+
+- Addon Developers
+  + Addons now have `ember-disable-prototype-extensions` included by default,
+    this ensures add-ons are written in a way that works regardless of the
+    consumers protytpe extension preference
```
Comment (chadhietala) [inline CHANGELOG.md]: prototype

**Comment by chadhietala** [comment]
:ship: :shipit:

**Comment by abuiles** [comment]
We should mention the following too https://github.com/ember-cli/ember-cli/pull/3629#issuecomment-85568103

**Comment by stefanpenner** [comment]
> We should mention the following too #3629 (comment)

updated, can you confirm its ok?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,82 @@
...
+The following changes are required if you are upgrading from the previous
+version:
+
+- Users
+  + updated to ember 1.11.0
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + `ember init` once again works inside an addon.
+  + error live-reloading now actually works!
+  + npm WARN for `makeError` and `tmpl` have been fixed
+  + ember-qunit was updated from `0.2.8` -> `0.3.0`, `this.render()` in a test now no-longer turns a jQuery object.
```
Comment (jbrown) [inline CHANGELOG.md]: returns*

**Pull request merged**